rpcs3/Emu/Memory/vm_locking.h

```diff
@@ -37,9 +37,10 @@ namespace vm
 	// Register range lock for further use
 	atomic_t<u64, 64>* alloc_range_lock();
 
-	void range_lock_internal(atomic_t<u64, 64>* range_lock, u32 begin, u32 size);
+	void range_lock_internal(atomic_t<u64, 64>* range_lock, u32 begin, u32 size, bool touch_mem);
 
 	// Lock memory range
+	template <bool touch_mem = true>
```

Comment:
if I'm not mistaken, Neko fixed similar lines to use upper CamelCase for template vars.

Reply:
noone cares.

Reply:
I care, that's why I fix template arg style when I see it. But that's just a nitpick.